.github/workflows/haskell.yml

```diff
@@ -94,6 +94,15 @@ jobs:
             cabal_version: 3.6.2.0
             cabal_project: cabal.project
             ignore_error: false
+          - name: 9.4.2-fusion-inspection-Werror
```

Comment:
Can you move this to line 81, next to the other 9.4.2 CI. Also, we can consider converting the existing 9.4.2 CI to fusion-plugin CI to avoid too many CIs.